### ISSUE TITLE
[EE] No screen found on Xorg after upgrade to kernel 4.10.3

### ISSUE THREAD

**Issue by HikvIneH** [Bumblebee-Project/Bumblebee#852]
All thread already solved this error, but this error happened after update to kernel 4.10.
Distro : Arch Linux 
Kernel : x86_64 Linux 4.10.3-1-ARCH
bumblebee version: 3.2.1
Nvidia version: 378.13

I've tried to append `rcutree.rcu_idle_gp_delay=1` in the bootloader. but still get the same error. 

and I think the problem is with the X since running `optirun --no-xorg glxsphere` seems to work.
and `modprobe nvidia` gave no errors.
and `lsmod | grep nvidia` will show nvidia aswell.

dmesg
```
[   48.666032] nvidia: module license 'NVIDIA' taints kernel.
[   48.666034] Disabling lock debugging due to kernel taint
[   48.677276] nvidia 0000:04:00.0: enabling device (0006 -> 0007)
[   48.677527] nvidia-nvlink: Nvlink Core is being initialized, major device number 243
[   48.677555] NVRM: loading NVIDIA UNIX x86_64 Kernel Module  378.13  Tue Feb  7 20:10:06 PST 2017
[   48.677556]  (using threaded interrupts)

[   48.749128] NVRM: failed to copy vbios to system memory.
[   48.749266] NVRM: RmInitAdapter failed! (0x30:0xffff:654)
[   48.749289] NVRM: rm_init_adapter failed for device bearing minor number 0
[  952.168163] NVRM: failed to copy vbios to system memory.
[  952.168287] NVRM: RmInitAdapter failed! (0x30:0xffff:654)
[  952.168307] NVRM: rm_init_adapter failed for device bearing minor number 0
[ 1297.020991] NVRM: failed to copy vbios to system memory.
[ 1297.021129] NVRM: RmInitAdapter failed! (0x30:0xffff:654)
[ 1297.021154] NVRM: rm_init_adapter failed for device bearing minor number 0
[ 1521.463179] NVRM: failed to copy vbios to system memory.
[ 1521.463300] NVRM: RmInitAdapter failed! (0x30:0xffff:654)
[ 1521.463335] NVRM: rm_init_adapter failed for device bearing minor number 0
[ 1802.179629] NVRM: failed to copy vbios to system memory.
[ 1802.179830] NVRM: RmInitAdapter failed! (0x30:0xffff:654)
[ 1802.179876] NVRM: rm_init_adapter failed for device bearing minor number 0
```

Xorg Log file
``````
[   541.841] 
X.Org X Server 1.19.2
Release Date: 2017-03-02
[   541.841] X Protocol Version 11, Revision 0
[   541.841] Build Operating System: Linux 4.9.13-1-lts x86_64 
[   541.841] Current Operating System: Linux challenger 4.10.3-1-ARCH #1 SMP PREEMPT Wed Mar 15 09:17:17 CET 2017 x86_64
[   541.841] Kernel command line: initrd=\intel-ucode.img initrd=\initramfs-linux.img root=PARTUUID=acb6c045-b9a7-4748-83c7-939060358d29 rw
[   541.841] Build Date: 03 March 2017  06:00:24PM
[   541.841]  
[   541.841] Current version of pixman: 0.34.0
[   541.841] 	Before reporting problems, check http://wiki.x.org
	to make sure that you have the latest version.
[   541.841] Markers: (--) probed, (**) from config file, (==) default setting,
	(++) from command line, (!!) notice, (II) informational,
	(WW) warning, (EE) error, (NI) not implemented, (??) unknown.
[   541.841] (==) Log file: "/var/log/Xorg.8.log", Time: Fri Mar 17 19:10:51 2017
[   541.841] (++) Using config file: "/etc/bumblebee/xorg.conf.nvidia"
[   541.841] (++) Using config directory: "/etc/bumblebee/xorg.conf.d"
[   541.841] (==) Using system config directory "/usr/share/X11/xorg.conf.d"
[   541.841] (==) ServerLayout "Layout0"
[   541.841] (==) No screen section available. Using defaults.
[   541.841] (**) |-->Screen "Default Screen Section" (0)
[   541.841] (**) |   |-->Monitor "<default monitor>"
[   541.842] (==) No device specified for screen "Default Screen Section".
	Using the first device section listed.
[   541.842] (**) |   |-->Device "DiscreteNvidia"
[   541.842] (==) No monitor specified for screen "Default Screen Section".
	Using a default monitor configuration.
[   541.842] (**) Option "AutoAddDevices" "false"
[   541.842] (**) Option "AutoAddGPU" "false"
[   541.842] (**) Not automatically adding devices
[   541.842] (==) Automatically enabling devices
[   541.842] (**) Not automatically adding GPU devices
[   541.842] (==) Max clients allowed: 256, resource mask: 0x1fffff
[   541.842] (WW) The directory "/usr/share/fonts/Type1/" does not exist.
[   541.842] 	Entry deleted from font path.
[   541.842] (WW) `fonts.dir' not found (or not valid) in "/usr/share/fonts/100dpi/".
[   541.842] 	Entry deleted from font path.
[   541.842] 	(Run 'mkfontdir' on "/usr/share/fonts/100dpi/").
[   541.842] (WW) `fonts.dir' not found (or not valid) in "/usr/share/fonts/75dpi/".
[   541.842] 	Entry deleted from font path.
[   541.842] 	(Run 'mkfontdir' on "/usr/share/fonts/75dpi/").
[   541.842] (==) FontPath set to:
	/usr/share/fonts/misc/,
	/usr/share/fonts/TTF/,
	/usr/share/fonts/OTF/
[   541.842] (++) ModulePath set to "/usr/lib/nvidia/xorg,/usr/lib/xorg/modules"
[   541.842] (==) |-->Input Device "<default pointer>"
[   541.842] (==) |-->Input Device "<default keyboard>"
[   541.842] (==) The core pointer device wasn't specified explicitly in the layout.
	Using the default mouse configuration.
[   541.842] (==) The core keyboard device wasn't specified explicitly in the layout.
	Using the default keyboard configuration.
[   541.842] (II) Loader magic: 0x822d60
[   541.842] (II) Module ABI versions:
[   541.842] 	X.Org ANSI C Emulation: 0.4
[   541.842] 	X.Org Video Driver: 23.0
[   541.842] 	X.Org XInput driver : 24.1
[   541.842] 	X.Org Server Extension : 10.0
[   541.842] (--) using VT number 2

[   541.843] (II) systemd-logind: logind integration requires -keeptty and -keeptty was not provided, disabling logind integration
[   541.843] (II) xfree86: Adding drm device (/dev/dri/card0)
[   541.843] (EE) /dev/dri/card0: failed to set DRM interface version 1.4: Permission denied
[   541.844] (--) PCI:*(0:4:0:0) 10de:0fe4:1028:05fa rev 161, Mem @ 0xe2000000/16777216, 0xd0000000/268435456, 0xe0000000/33554432, I/O @ 0x00003000/128
[   541.844] (WW) Open ACPI failed (/var/run/acpid.socket) (No such file or directory)
[   541.844] (II) LoadModule: "glx"
[   541.844] (II) Loading /usr/lib/nvidia/xorg/libglx.so
[   541.847] (II) Module glx: vendor="NVIDIA Corporation"
[   541.847] 	compiled for 4.0.2, module version = 1.0.0
[   541.848] 	Module class: X.Org Server Extension
[   541.848] (II) NVIDIA GLX Module  378.13  Tue Feb  7 18:25:34 PST 2017
[   541.848] (II) LoadModule: "nvidia"
[   541.848] (II) Loading /usr/lib/xorg/modules/drivers/nvidia_drv.so
[   541.848] (II) Module nvidia: vendor="NVIDIA Corporation"
[   541.848] 	compiled for 4.0.2, module version = 1.0.0
[   541.848] 	Module class: X.Org Video Driver
[   541.848] (II) LoadModule: "mouse"
[   541.848] (II) Loading /usr/lib/xorg/modules/input/mouse_drv.so
[   541.848] (II) Module mouse: vendor="X.Org Foundation"
[   541.848] 	compiled for 1.19.0, module version = 1.9.2
[   541.848] 	Module class: X.Org XInput Driver
[   541.848] 	ABI class: X.Org XInput driver, version 24.1
[   541.848] (II) LoadModule: "kbd"
[   541.848] (WW) Warning, couldn't open module kbd
[   541.848] (II) UnloadModule: "kbd"
[   541.848] (II) Unloading kbd
[   541.848] (EE) Failed to load module "kbd" (module does not exist, 0)
[   541.848] (II) NVIDIA dlloader X Driver  378.13  Tue Feb  7 18:01:51 PST 2017
[   541.848] (II) NVIDIA Unified Driver for all Supported NVIDIA GPUs
[   541.848] (II) Loading sub module "fb"
[   541.848] (II) LoadModule: "fb"
[   541.848] (II) Loading /usr/lib/xorg/modules/libfb.so
[   541.848] (II) Module fb: vendor="X.Org Foundation"
[   541.848] 	compiled for 1.19.2, module version = 1.0.0
[   541.848] 	ABI class: X.Org ANSI C Emulation, version 0.4
[   541.848] (II) Loading sub module "wfb"
[   541.848] (II) LoadModule: "wfb"
[   541.849] (II) Loading /usr/lib/xorg/modules/libwfb.so
[   541.849] (II) Module wfb: vendor="X.Org Foundation"
[   541.849] 	compiled for 1.19.2, module version = 1.0.0
[   541.849] 	ABI class: X.Org ANSI C Emulation, version 0.4
[   541.849] (II) Loading sub module "ramdac"
[   541.849] (II) LoadModule: "ramdac"
[   541.849] (II) Module "ramdac" already built-in
[   541.849] (II) NVIDIA(0): Creating default Display subsection in Screen section
	"Default Screen Section" for depth/fbbpp 24/32
[   541.849] (==) NVIDIA(0): Depth 24, (==) framebuffer bpp 32
[   541.849] (==) NVIDIA(0): RGB weight 888
[   541.849] (==) NVIDIA(0): Default visual is TrueColor
[   541.849] (==) NVIDIA(0): Using gamma correction (1.0, 1.0, 1.0)
[   541.849] (**) NVIDIA(0): Option "NoLogo" "true"
[   541.849] (**) NVIDIA(0): Option "ProbeAllGpus" "false"
[   541.849] (**) NVIDIA(0): Option "UseEDID" "false"
[   541.849] (**) NVIDIA(0): Option "UseDisplayDevice" "none"
[   541.849] (**) NVIDIA(0): Enabling 2D acceleration
[   541.849] (**) NVIDIA(0): Ignoring EDIDs
[   541.849] (**) NVIDIA(0): Option "UseDisplayDevice" set to "none"; enabling NoScanout
[   541.849] (**) NVIDIA(0):     mode
[   541.852] (EE) NVIDIA(GPU-0): Failed to initialize the NVIDIA GPU at PCI:4:0:0.  Please
[   541.852] (EE) NVIDIA(GPU-0):     check your system's kernel log for additional error
[   541.852] (EE) NVIDIA(GPU-0):     messages and refer to Chapter 8: Common Problems in the
[   541.852] (EE) NVIDIA(GPU-0):     README for additional information.
[   541.852] (EE) NVIDIA(GPU-0): Failed to initialize the NVIDIA graphics device!
[   541.852] (EE) NVIDIA(0): Failing initialization of X screen 0
[   541.852] (II) UnloadModule: "nvidia"
[   541.852] (II) UnloadSubModule: "wfb"
[   541.852] (II) UnloadSubModule: "fb"
[   541.852] (EE) Screen(s) found, but none have a usable configuration.
[   541.852] (EE) 
Fatal server error:
[   541.852] (EE) no screens found(EE) 
[   541.852] (EE) 
Please consult the The X.Org Foundation support 
	 at http://wiki.x.org
 for help. 
[   541.852] (EE) Please also check the log file at "/var/log/Xorg.8.log" for additional information.
[   541.852] (EE) 
[   541.852] (EE) Server terminated with error (1). Closing log file.
``````

Any help would will be really apreciated!

**Comment by HikvIneH** [comment]
Solved the problem by downgrading the lts kernel (linux-lts 4.9). thank you everyone!

**Comment by HikvIneH** [comment]
it is not solved actually in kernel 4.10 I still have the same output.
